content/cpp/concepts/constructors/constructors.md

```diff
@@ -16,75 +16,121 @@ CatalogContent:
-    // Define a constructor that outputs a message
-    MyClass() {
-      std::cout << "The constructor was executed!";
+    // Define a basic constructor
+    MyClass () {
+      std::cout << "The constructor was executed!" << std::endl;
     }
 };
 
 int main() {
-  MyClass myObj;  // This calls the constructor
+  MyClass myObject;
   return 0;
 }
 ```
 
-Using arguments in a constructor:
+The result of the following code:
 
 ```cpp
-class Circle {
+The constructor was executed!
+```
+
+- Parameterized Constructor
+  A parameterized constructor is one that accepts arguments, allowing initial values to be set for an object's attributes at the time of creation. This constructor is useful for initializing objects with custom data or values that differ between instances.
+
+```cpp
+class Rectangle {
   public:
-    int X;
-    int Y;
-    int radius;
+    int width;
+    int height;
 
-    Circle (int a, int b, int c) {
-      X = a;
-      Y = b;
-      radius = c;
+    // Declare the constructor
+    Rectangle (int w, int h) {
+      width = w;
+      height = h;
     }
 };
 
 int main() {
-  // Call the constructor using arguments
-  Circle myCircle1(5,5,10);
-  Circle myCircle2(0,0,5);
+  // Declare implicitly
+  Rectangle rect1(10, 20);
+  // Declare explicitly
+  Rectangle rect2 = Rectangle(5, 15);
+
+  std::cout << "Rectangle 1: Width = " << rect1.width << ", Height = " << rect1.height << std::endl;
+  std::cout << "Rectangle 2: Width = " << rect2.width << ", Height = " << rect2.height << std::endl;
+}
+```
+
+The result of the following code:
 
-  // Print out the attribute values set for each object
+```cpp
+Rectangle 1: Width = 10, Height = 20
+Rectangle 2: Width = 5, Height = 15
+```
 
-  std::cout << "X=" << myCircle1.X << ", Y=" << myCircle1.Y << ", radius=" << myCircle1.radius << "\n";
-  // Output: X=5, Y=5, radius=10
+## Codebyte Example
 
-  std::cout << "X=" << myCircle2.X << ", Y=" << myCircle2.Y << ", radius=" << myCircle2.radius << "\n";
-  // Output: X=0, Y=0, radius=5
+Run the codebyte example below to see how constructors work:
 
-  return 0;
+- Basic Constructor
+
+```codebyte/cpp
+#include <iostream>
+
+class Circle {
+  public:
+    int radius;
+
+  Circle() {
+    radius = 5;
+  }
+};
+
+int main() {
+  Circle circle1;
+
+  std::cout << "The radius of the circle1 is " << circle1.radius << std::endl;
 }
 ```
 
-Like member functions, once declared in the class, the constructor can be defined outside the class:
+- Parameterized Constructor
+
+```codebyte/cpp
+#include <iostream>
 
-```cpp
 class Circle {
   public:
-    int X;
-    int Y;
     int radius;
 
-    // Declare the constructor
-    Circle (int a, int b, int c);
+    Circle(int r) {
+      radius = r;
+    }
 };
 
-// Define constructor outside the class
-Circle::Circle(int a, int b, int c) {
-  X = a;
-  Y = b;
-  radius = c;
-};
+int main() {
+  Circle circle2(11);
+  std::cout << "The radius of the circle2 is " << circle2.radius << std::endl;
+}
 ```
```

Comment:
There should be only one codebyte example unless we have something very different to show in both examples as per the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md). 

Please remove the basic constructor code from codebyte example.